canonical-web-and-design/maas-ui
# Backport network filters, IP table and analytics

## Done
Backported the following three commits into the legacy app:
- https://git.launchpad.net/maas/commit/?id=e5c49174c00da13058b997ae4479e80fecb9b3cd
- https://git.launchpad.net/maas/commit/?id=57df4a43aa9d73f1f50904a5bf640a7300cabcf2
- https://git.launchpad.net/maas/diff/?id=48ecbab211c40720ba4d42d683b56f44390a2a4d

## QA
- Check the link speed filters work on the machines list page
- Check the subnets IP table looks the same as the launchpad master
- Click around and see that links and buttons still work
